packages/react-devtools-shared/src/backend/ReactSymbols.js

```diff
@@ -19,6 +19,9 @@ export const CONCURRENT_MODE_SYMBOL_STRING = 'Symbol(react.concurrent_mode)';
 export const CONTEXT_NUMBER = 0xeace;
 export const CONTEXT_SYMBOL_STRING = 'Symbol(react.context)';
 
+export const SERVER_CONTEXT_NUMBER = 0xeacf;
```

Comment:
These don't exist anywhere since we removed the numbers before we published anything. So you can remove the number cases.